api/controllers/routes/agency.js

```diff
@@ -0,0 +1,34 @@
+const express = require("express");
+const app = express();
+const requestUtils = require("../../utils/request");
+const queryUtils = require("../../utils/query");
+
+const env = process.env.NODE_ENV || 'development';
```

Comment:
Can we make this a global const? Maybe the ones above as well? They are used on all routes so it would make sense, again if possible.

Reply:
I'm generally against global constants unless it is intended to be a single value for all scenarios, and this is pulling from the environment so I don't know if that's the right application. Someone with more knowledge of node might know a better way of doing this.

Reply:
I could, but this is just importing files like any other language would, so I'd be making a global constant for imports, which seems redundant.

Reply:
Yes but they would all reference the same thing which is what you are doing now but having to set it up per page and thus repeating yourself. It is like having a static class in Java that returns item or one setup method for a group of tests that need all of the same setup material.